### DISCUSSION TITLE
【求助】俺的图图呢？

### DISCUSSION THREAD

**Discussion by user1-cloud** [share121/inter-knot#670] [其他]
![image](https://github.com/user-attachments/assets/33ba171d-7306-41a7-98a6-19868db6ce26)
话说PC端没有必要只分三列吧━┳━　━┳━（原来是屏幕大小自适应）

**Comment by lllliufeng** [comment]
这下六列了

**Comment by lllliufeng** [comment]
原来是自适应屏幕大小(￣▽￣)／

**Comment by user1-cloud** [comment]
> 原来是自适应屏幕大小(￣▽￣)／
还真是
